src/test/java/duke/EventTest.java

```diff
@@ -0,0 +1,15 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+
+    Task task = new Event("read book", "31 Dec 2020");
```

Comment:
Maybe you should make this task private since it's not going to be accessed outside this class.